tests/e2e/wdio.conf.js

```diff
@@ -85,9 +86,9 @@ export function buildForChrome() {
 }
 
 export const config = {
-  specs: argv.debug ? [['**/*.spec.js']] : ['**/*.spec.js'],
+  specs: [['**/*.spec.js']],
   specFileRetries: 2,
```

Comment:
There are still bugs in Firefox implementation, and sometimes we have navigation errors (which are coused by the Firefox itself). I recommend to keep it. However, we can test it out, how it will go without it, and if it is required, we can bring it back.